cuml/dbscan_demo.ipynb

```diff
@@ -51,40 +52,34 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "n_samples = 100000\n",
-    "n_features = 128\n",
+    "n_samples = 10**4\n",
+    "n_features = 2\n",
     "\n",
-    "eps = 3\n",
-    "min_samples = 2"
+    "eps = 0.15\n",
+    "min_samples = 3\n",
+    "random_state = 23"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Generate Data\n",
-    "\n",
-    "### Host"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "host_data, host_labels = make_blobs(\n",
-    "   n_samples=n_samples, n_features=n_features, centers=5, random_state=7)\n",
-    "\n",
-    "host_data = pd.DataFrame(host_data)\n",
-    "host_labels = pd.Series(host_labels)"
+    "## Generate Data"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Device"
+    "%%time\n",
```

Comment:
it needs to be in a seperate cell, it is being treated like a part of the markdown cell